tlatools/org.lamport.tlatools/src/tlc2/tool/liveness/TableauNodePtrTable.java

```diff
@@ -131,11 +131,19 @@ public final long get(final long k, final int tidx) {
 		}
 	}
 
+	public final void put(final long k, final int tidx) {
+		put(k, tidx, UNDONE, UNDONE - 1);

```

Comment:
Is there any significance to the value `UNDONE - 1`, or is it just a value that is neither `UNDONE` nor any other legal value?

Reply:
Added a comment.

Reply:
https://github.com/tlaplus/tlaplus/issues/990 is about the main part of the question.